WordPress/Classes/Models/Post+CoreDataProperties.swift

```diff
@@ -4,13 +4,16 @@ import CoreData
 extension Post {
 
     @NSManaged var commentCount: NSNumber?
+    @NSManaged var disabledPublicizeConnections: Dictionary<NSNumber, Dictionary<String, String>>?
```

Comment:
None of us is proficient in Swift... since it's mutating all of the time! Thanks for updating this one!